CONTRIBUTING.md

```diff
@@ -313,3 +313,5 @@ python -m http.server <port>
 ```
 
 Then open the browser at `0.0.0.0:<port>` (e.g. `0.0.0.0:1234`) and click to `html` folder.
+
```

Comment:
Maybe we could just replace `0.0.0.0` by `localhost` and avoid a specific note about WSL ?

Reply:
Yeah, `localhost` seems more concise. Okay, I'll change it to that.

Reply:
Yeah, `localhost` makes more sense. I'll make the change then.

Reply:
Done!